source/layout/theme.liquid

```diff
@@ -31,7 +31,7 @@
 
     <div class="grid">
 
-      <main class="site__main">
+      <main class="site__main" aria-label="content">
```

Comment:
What's the difference between `<main role="content">` and `<main aria-label="content">`?
As far as I know they are pretty much the same. Because of the semantic `<main>` tag, aria-role can be omitted.